geopoint/challenges/default.json

```diff
@@ -79,7 +79,7 @@
           "operation": "distanceRange",
           "warmup-iterations": 200,
           "iterations": 100,
-          "target-throughput": 0.6
+          "target-throughput": 0.5
```

Comment:
should we use `target-interval: 2`?

Reply:
I was really considering it but I thought that it might be trappy here as it can easily be overlooked.